qa/integration-tests/src/test/java/io/zeebe/broker/it/SecurityTest.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.broker.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.grpc.netty.GrpcSslContexts;
+import io.netty.handler.ssl.SslContext;
+import io.zeebe.broker.Loggers;
+import io.zeebe.broker.it.clustering.ClusteringRule;
+import io.zeebe.client.ZeebeClientBuilder;
+import io.zeebe.client.api.response.Topology;
+import io.zeebe.gateway.impl.configuration.GatewayCfg;
+import javax.net.ssl.SSLException;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.RuleChain;
+import org.junit.rules.Timeout;
+
+public class SecurityTest {
+  public Timeout testTimeout = Timeout.seconds(120);
+
+  public ClusteringRule clusteringRule =
+      new ClusteringRule(
+          3, 3, 3, brokerCfg -> {}, this::configureGatewayForTls, this::configureClientForTls);
+
+  public GrpcClientRule clientRule =
+      new GrpcClientRule(
+          cfg ->
+              configureClientForTls(
+                  cfg.brokerContactPoint(clusteringRule.getGatewayAddress().toString())));
+
+  @Rule
+  public RuleChain ruleChain =
+      RuleChain.outerRule(testTimeout).around(clusteringRule).around(clientRule);
+
+  @Test
+  public void shouldEstablishSecureConnection() {
+    // when
+    final Topology topology = clientRule.getClient().newTopologyRequest().send().join();
+
+    // then
+    assertThat(topology.getBrokers().size()).isEqualTo(3);
+  }
+
+  private SslContext createContext() {
+    try {
+      return GrpcSslContexts.forClient()
+          .trustManager(
+              io.zeebe.broker.it.clustering.DeploymentClusteredTest.class
+                  .getClassLoader()
+                  .getResourceAsStream("security/ca.cert.pem"))
+          .build();
+    } catch (SSLException e) {
+      Loggers.SYSTEM_LOGGER.warn("Expected to create SSL context during test but failed with: ", e);
```

Comment:
Why not let the exception bubble up? It's a test, and we probably want to fail the test if we didn't get the context as expected?